app/frontend/src/features/profile/ProfileForm.tsx

```diff
@@ -6,18 +6,20 @@ import { Controller, useForm } from "react-hook-form";
 import Alert from "../../components/Alert";
 import Button from "../../components/Button";
 import TextField from "../../components/TextField";
-import { ProfileFormData } from "../../service/user";
-import ProfileTextInput from "./ProfileTextInput";
 import { updateUserProfile } from "./index";
+import ProfileTextInput from "./ProfileTextInput";
+import { ProfileFormData } from "../../service/user";
+import { theme } from "../../theme";
 import { useAppDispatch, useTypedSelector } from "../../store";
 
 const useStyles = makeStyles({
   buttonContainer: {
     display: "flex",
+    paddingTop: theme.spacing(1),
```

Comment:
Do you think it's better to use `makeStyles((theme) => {`? Then you don't need to import. The drawback is, then you can't use it in the component (say, if you use a `<Box>`).

Reply:
That's a good suggestion. I think there's actually a more official `useTheme` hook so maybe should use that instead 🤔